packages/css/src/tokens.json

```
@@ -503,6 +503,34 @@
       "category": "layers",
       "value": "rgba(0, 0, 0, 0.2) 0px 2px 16px 0px",
       "type": "number"
+    },
+   {
+      "theme": "maize",
+      "name": "opacity_75",
```

We should stick with dashes rather than underscores to match the style of the other token names.
```suggestion
      "name": "opacity-75",
```